app/javascript/utils/distributions_and_transfers.js

```diff
@@ -106,3 +106,58 @@ $(function() {
     request_storage_location_and_populate_item(default_item);
   });
 });
+
+$(document).ready(function (e) {
+  $(document).on('change', '.dist_delivery_method', hide_show_shipping_cost);
+  show_shipping_cost();
+});
+
+function show_shipping_cost()
```

Comment:
Can we use a Stimulus controller for this behavior? It's a lot cleaner than jQuery DOM munging.

Reply:
done